frontend/src/metabase-types/api/card.ts

```diff
@@ -82,7 +82,11 @@ export type CardDisplayType =
   | "map"
   | "scatter"
   | "waterfall"
-  | (string & unknown);
+  | "action"
```

Comment:
Any suggestions on how else would we do it besides combining them? 

A lot of our tests use `createMockCard` with the `VirtualCardDisplay` types, and when we use `registerVisualizations` we register all of the visualizations together (so `CardDisplayType` and `VirtualCardDisplay`) so they're all in a map together (so the final type of the visualizations object is `Record<CardDisplayType | VirtualCardDisplay, Visualization>`.

I would agree that we should be treating these differently since there's a difference between a data-based card and cosmetic cards. _But_ based on the codebase and how we treat these visualizations I'm not totally sure we have any other option without doing a refactor

Reply:
For now, I'll combine them. I think it's a better state than the one we were in before, but I don't think we should do anything else beyond that

Reply:
that's fair. I haven't tried to fix this, but at a glance, we should only use `createMockCard` for actual cards which shouldn't contain any of these `action`, `heading`, etc. So the type would work. Maybe there other places we still need to fix though.

Let's go with your assessment this time.